packages/kbn-apm-synthtrace/README.md

```diff
@@ -128,12 +139,11 @@ Note:
 
 ### Scenario options
 
-| Option           | Type      | Default | Description                          |
-| ---------------- | --------- | :------ | ------------------------------------ |
-| `--from`         | [date]    | `now()` | The start of the time window         |
-| `--to`           | [date]    |         | The end of the time window           |
-| `--live`         | [boolean] |         | Generate and index data continuously |
-| `--scenarioOpts` |           |         | Raw options specific to the scenario |
```

Comment:
Would be have an alternative to run with dynamic specific options for the scenario? or is it needed now that we define 100% of the options in the code?

Reply:
@yngrdyn I did not understand the comment. As part if this PR, i have only reformatted the table based on my IDE recommending a better formatting

Reply:
Good catch.